doc/package-notes.xml

```diff
@@ -464,7 +464,6 @@ it. Place the resulting <filename>package.nix</filename> file into
     or because they cannot be patched.The steps to launch a game 
     directly are: 
     <orderedlist>
```

Comment:
Done, removed the whole section.
Please tell me if there is any change still to be made.
Thanks!